Tests fail after 2025-12-31

While working on [reproducible builds](https://reproducible-builds.org/) for [openSUSE](https://en.opensuse.org/openSUSE:Reproducible_Builds), I found that
our `python-dateparser` 1.2.2 package fails its tests when run after 2026-01-01T01:06:13

```
 self = <tests.test_search.TestTranslateSearch testMethod=test_search_dates_with_prepositions>

     def test_search_dates_with_prepositions(self):
         """Test `search_dates` for parsing Russian date ranges with prepositions and language detection."""
         result = search_dates(
             "Сервис будет недоступен с 12 января по 30 апреля.",
             add_detected_language=True,
             languages=["ru"],
         )
         expected = [
             ("12 января", datetime.datetime(2025, 1, 12, 0, 0), "ru"),
             ("30 апреля", datetime.datetime(2025, 4, 30, 0, 0), "ru"),
         ]   
 >       assert result == expected
 E       AssertionError: assert [('12 января'... 0, 0), 'ru')] == [('12 января'... 0, 0), 'ru')]
 E       
 E         At index 0 diff: ('12 января', datetime.datetime(2041, 1, 12, 0, 0), 'ru') != ('12 января', datetime.datetime(2025, 1, 12, 0, 0), 'ru')
 E           
 E         Full diff:
 E           [
 E               (
 E                   '12 января',...
 E         
 E         ...Full output truncated (15 lines hidden), use '-vv' to show
```

Tests should keep working in the future. e.g. you could mock the current time.

It is now past 2025-12-31. As predicted. 

```python
[   47s] =================================== FAILURES ===================================
[   47s] ___________ TestTranslateSearch.test_search_dates_with_prepositions ____________
[   47s]
[   47s] self = <tests.test_search.TestTranslateSearch testMethod=test_search_dates_with_prepositions>
[   47s]
[   47s]     def test_search_dates_with_prepositions(self):
[   47s]         """Test `search_dates` for parsing Russian date ranges with prepositions and language detection."""
[   47s]         result = search_dates(
[   47s]             "Сервис будет недоступен с 12 января по 30 апреля.",
[   47s]             add_detected_language=True,
[   47s]             languages=["ru"],
[   47s]         )
[   47s]         expected = [
[   47s]             ("12 января", datetime.datetime(2025, 1, 12, 0, 0), "ru"),
[   47s]             ("30 апреля", datetime.datetime(2025, 4, 30, 0, 0), "ru"),
[   47s]         ]
[   47s] >       assert result == expected
[   47s] E       AssertionError: assert [('12 января', datetime.datetime(2026, 1, 12, 0, 0), 'ru'), ('30 апреля', datetime.datetime(2026, 4, 30, 0, 0), 'ru')] == [('12 января', datetime.datetime(2025, 1, 12, 0, 0), 'ru'), ('30 апреля', datetime.datetime(2025, 4, 30, 0, 0), 'ru')]
[   47s] E
[   47s] E         At index 0 diff: ('12 января', datetime.datetime(2026, 1, 12, 0, 0), 'ru') != ('12 января', datetime.datetime(2025, 1, 12, 0, 0), 'ru')
[   47s] E
[   47s] E         Full diff:
[   47s] E           [
[   47s] E               (
[   47s] E                   '12 января',
[   47s] E         -         datetime.datetime(2025, 1, 12, 0, 0),
[   47s] E         ?                              ^
[   47s] E         +         datetime.datetime(2026, 1, 12, 0, 0),
[   47s] E         ?                              ^
[   47s] E                   'ru',
[   47s] E               ),
[   47s] E               (
[   47s] E                   '30 апреля',
[   47s] E         -         datetime.datetime(2025, 4, 30, 0, 0),
[   47s] E         ?                              ^
[   47s] E         +         datetime.datetime(2026, 4, 30, 0, 0),
[   47s] E         ?                              ^
[   47s] E                   'ru',
[   47s] E               ),
[   47s] E           ]
[   47s]
[   47s] tests/test_search.py:1105: AssertionError
[   47s] =========================== short test summary info ============================
[   47s] FAILED tests/test_search.py::TestTranslateSearch::test_search_dates_with_prepositions - AssertionError: assert [('12 января', datetime.datetime(2026, 1, 12, 0, 0), 'ru'), ('30 апреля', datetime.datetime(2026, 4, 30, 0, 0), 'ru')] == [('12 января', datetime.datetime(2025, 1, 12, 0, 0), 'ru'), ('30 апреля', datetime.datetime(2025, 4, 30, 0, 0), 'ru')]
[   47s]
[   47s]   At index 0 diff: ('12 января', datetime.datetime(2026, 1, 12, 0, 0), 'ru') != ('12 января', datetime.datetime(2025, 1, 12, 0, 0), 'ru')
[   47s]
[   47s]   Full diff:
[   47s]     [
[   47s]         (
[   47s]             '12 января',
[   47s]   -         datetime.datetime(2025, 1, 12, 0, 0),
[   47s]   ?                              ^
[   47s]   +         datetime.datetime(2026, 1, 12, 0, 0),
[   47s]   ?                              ^
[   47s]             'ru',
[   47s]         ),
[   47s]         (
[   47s]             '30 апреля',
[   47s]   -         datetime.datetime(2025, 4, 30, 0, 0),
[   47s]   ?                              ^
[   47s]   +         datetime.datetime(2026, 4, 30, 0, 0),
[   47s]   ?                              ^
[   47s]             'ru',
[   47s]         ),
[   47s]     ]
[   47s] ========== 1 failed, 23920 passed, 16 skipped, 5 deselected in 31.44s ==========
```

Deepl.com says that ""Сервис будет недоступен с 12 января по 30 апреля." translates to "The service will be unavailable from January 12 to April 30.". How can you possibly expect a year from that?

--- a/tests/test_search.py
+++ b/tests/test_search.py
@@ -1100,8 +1100,8 @@ class TestTranslateSearch(BaseTestCase):
             languages=["ru"],
         )
         expected = [
-            ("12 января", datetime.datetime(2025, 1, 12, 0, 0), "ru"),
-            ("30 апреля", datetime.datetime(2025, 4, 30, 0, 0), "ru"),
+            ("12 января", datetime.datetime(today.year, 1, 12, 0, 0), "ru"),
+            ("30 апреля", datetime.datetime(today.year, 4, 30, 0, 0), "ru"),
         ]
         assert result == expected
 

